packages/sdk/src/application/ApplicationClient.ts

```diff
@@ -0,0 +1,411 @@
+import { getInstillAdditionalHeaders, getQueryString } from "../helper";
+import { APIResource } from "../main/resource";
+import {
+  Application,
+  ChatRequest,
+  ChatResponse,
+  CreateApplicationRequest,
+  CreateApplicationResponse,
+  CreateConversationRequest,
+  CreateConversationResponse,
+  CreateMessageRequest,
+  CreateMessageResponse,
+  DeleteApplicationRequest,
+  DeleteConversationRequest,
+  DeleteMessageRequest,
+  GetApplicationRequest,
+  GetApplicationResponse,
+  GetPlaygroundConversationRequest,
+  GetPlaygroundConversationResponse,
+  ListApplicationsRequest,
+  ListApplicationsResponse,
+  ListConversationsRequest,
+  ListConversationsResponse,
+  ListMessagesRequest,
+  ListMessagesResponse,
+  RestartPlaygroundConversationRequest,
+  RestartPlaygroundConversationResponse,
+  UpdateAIAssistantAppPlaygroundRequest,
+  UpdateAIAssistantAppPlaygroundResponse,
+  UpdateApplicationRequest,
+  UpdateApplicationResponse,
+  UpdateConversationRequest,
+  UpdateConversationResponse,
+  UpdateMessageRequest,
+  UpdateMessageResponse,
+} from "./types";
+
+export class ApplicationClient extends APIResource {
+  async listApplications(
+    props: ListApplicationsRequest & {
+      enablePagination?: boolean;
+    },
+  ): Promise<ListApplicationsResponse | Application[]> {
+    const { pageSize, pageToken, view, enablePagination, ownerId } = props;
+
+    try {
+      const queryString = getQueryString({
+        baseURL: `/v1alpha/namespaces/${ownerId}/apps`,
+        pageSize: pageSize ?? undefined,
+        pageToken: pageToken ?? undefined,
+        view: view ?? undefined,
```

Comment:
@thewbuk you can set the parameter as undefined so you don't need to shortcut it here